charts/retool/values.yaml

```diff
@@ -231,7 +231,7 @@ extraVolumeMounts: []
 
 extraVolumes: []
 
-# These resource specifications apply to the main backend and workflows backend pods.
+# These resource specifications will apply to the main backend, jobs-runner, db-connector, and workflows backend pods(unless if container specific resources are set)
```

Comment:
nit, reword:

# These resource specifications will apply to the main backend, jobs-runner, dbconnector, and workflows-backend pods unless container specific resources are set.